spec/db.spec.js

```diff
@@ -2,22 +2,16 @@ import chai, { expect } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { db } from '../src';
 import config from './databases/config';
-import * as dbSpecHelper from './databases/helper';
 chai.use(chaiAsPromised);
 
-
 /**
  * List of supported DB clients.
  * The "integration" tests will be executed for all supported DB clients.
  * And ensure all these clients has the same API and output results.
  */
-const SUPPORTED_DB_CLIENTS = ['mysql', 'postgresql'];
-
-
+const SUPPORTED_DB_CLIENTS = ['mysql', 'postgresql', 'sqlserver'];
```

Comment:
Please keep 2 break lines between the code in the file's root level. Is just a style we already used to. I'm going to include it in the contributing guide (that does not exist yet).